utils/spdx/src/test/kotlin/SpdxExpressionParserTest.kt

```diff
@@ -176,32 +176,31 @@ class SpdxExpressionParserTest : WordSpec({
                 SpdxExpression.parse("license WITH exception+")
             }
 
-            exception.message shouldBe "extraneous input '+' expecting <EOF>"
+            exception.message shouldBe "Unexpected token 'PLUS(position=23)'."
```

Comment:
IMO this reads a bit odd with the position being part of the quoted token. Instead, I'd expect something like "Unexpected PLUS token at position 23." (I'd also not quote 'PLUS' as that's not the literal character, or use '+', similar to line 195.)

But I'd be fine with improving this after the merge. Similar below in line 187.

Reply:
As discussed in the core-dev meeting I will make a follow-up PR that will improve the error messages.